### ISSUE TITLE
Project crashes after one timeline ends

### ISSUE THREAD

**Issue by obesecat** [coppolaemilio/dialogic#1657]
The get_layout_node() function (line 439 of the DialogicGameHandler.gd script) returns null and crashes the game instantly when attempting to open another timeline after the previous (first one) is completed, with the error "Trying to return a previously freed instance" in a 3d environment using the Dialogic.start() function

Changing the timeline that first plays and whether the second timeline is the same as the first or different led to the same result

 - OS: Windows 11
 - Godot Version: 4.1.1
 - Dialogic Version: 2.0

**Comment by Jowan-Spooner** [comment]
Thanks for the report. I will try to test this later today or tonight.

**Comment by obesecat** [comment]
Thanks, I uploaded a project with the same issue if you can't reproduce it https://github.com/obesecat/dialogicexample

Apologies in advance if it's something I'm doing wrong.

**Comment by Jowan-Spooner** [comment]
@obesecat I was able to reproduce the crash and I think I managed to add a simple fix. Maybe you could try it yourself (PR #1658). If you don't want to reinstall dialogic you can simply switch out the old DialogicGameHandler.get_layout_node() method for this:
```gdscript
func get_layout_node() -> Node:
	# `null` doesn't really work as a default for `get_meta`, because it'll
	# still throw an error if the meta entry doesn't exist. Revisit this if
	# Godot ever gives us a way to explicitly have `null` as a default.
	# (oddfacade 2023-07)
	var tree := get_tree()
	return (
		tree.get_meta('dialogic_layout_node')
		if tree.has_meta('dialogic_layout_node') and 
			is_instance_valid(tree.get_meta('dialogic_layout_node'))
		else null
	)
```

**Comment by gallsy** [comment]
I just tested the PR and can confirm it's fixed the issue for me in Godot 4.0.4

**Comment by Jowan-Spooner** [comment]
Glad to hear that. Then I will merge the PR. Thank you very much for the report. And good luck with your project!
